tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -329,45 +329,43 @@ describe('Home - Game List', () => {
     });
 
     it('Disables spectate button if on home view before game finishes', () => {
-      cy.skipOnGameStateApi();
       cy.signupOpponent(playerOne);
       cy.signupOpponent(playerTwo);
 
       // Navigate to homepage and select spectate tab
-      cy.visit('/');
+      // cy.visit('/');
       cy.get('[data-cy-game-list-selector=spectate]').click();
 
       // Finished by conceded
       cy.log('Setup game end by conceding');
       setupGameBetweenTwoUnseenPlayers('conceded');
-      cy.concedeOpponent();
-      cy.get('@concededGameId').then(function () {
-        cy.get(`[data-cy-spectate-game=${this.concededGameId}]`).should('be.disabled');
+      cy.get('@concededGameId').then((concededGameId) => {
+        cy.concedeOpponent(concededGameId);
+        cy.get(`[data-cy-spectate-game=${concededGameId}]`).should('be.disabled');
       });
       cy.log('Game ended by conceding - successfully disabled spectate button');
 
       // Navigate to homepage and select spectate tab
       cy.reload();
-      cy.get('[data-cy-game-list-selector=spectate]').click();
+      cy.url().should('include', '/spectate-list');
 
       // Finished by stalemate
       cy.log('Setup game end by stalemate');
       setupGameBetweenTwoUnseenPlayers('stalemate');
 
-      cy.stalemateOpponent();
-      cy.recoverSessionOpponent(playerOne);
-
+      
       // make sure button is still enabled after 1 person stalemates
-      cy.get('@stalemateGameId').then(function () {
-        cy.get(`[data-cy-spectate-game=${this.stalemateGameId}]`).should('be.not.disabled');
-        cy.acceptStalemateOpponent();
-        cy.get(`[data-cy-spectate-game=${this.stalemateGameId}]`).should('be.disabled');
+      cy.get('@stalemateGameId').then(function (stalemateGameId) {
+        cy.stalemateOpponent(stalemateGameId);
+        cy.recoverSessionOpponent(playerOne);
+        cy.get(`[data-cy-spectate-game=${stalemateGameId}]`).should('be.not.disabled');
+        cy.acceptStalemateOpponent(stalemateGameId);
```

Comment:
The GameState API gets `gameId` as an explicit request param rather than from the session, so to make a request on behalf of a user when the test-perspective user isn't in the game requires caching the game id beforehand and passing it explicitliy in the command like this.

Here our user is sitting on the home page while two users request + accept a stalemate